client/tools/mgr-virtualization/mgr-virtualization.spec

```diff
@@ -217,26 +238,56 @@ if [ -d /proc/xen ]; then
     # change the default template to the xen version
     sed -i 's@^IMAGE_CFG_TEMPLATE=/etc/sysconfig/rhn/kvm-template.xml@IMAGE_CFG_TEMPLATE=/etc/sysconfig/rhn/xen-template.xml@' /etc/sysconfig/rhn/image.cfg
 fi
+%if 0%{?default_py3}
+%service_add_post mgr-virtualization.timer
+
+%pre
+%service_add_pre mgr-virtualization.timer
```

Comment:
Is this really needed for pre and post?

Reply:
I think yes. If macros exist it is typically correct to use them.